charts/argo-cd/templates/argocd-application-controller/deployment.yaml

```
@@ -21,9 +21,9 @@ spec:
   template:
     metadata:
       annotations:
-        checksum/cmd-params: {{ include (print $.Template.BasePath "/argocd-configs/argocd-cmd-params-cm.yaml") . | sha256sum }}
+        checksum/cmd-params: {{ (include (print $.Template.BasePath "/argocd-configs/argocd-cmd-params-cm.yaml") . | fromYaml).data | toYaml | sha256sum }}
         {{- if .Values.configs.cm.create }}
-        checksum/cm: {{ include (print $.Template.BasePath "/argocd-configs/argocd-cm.yaml") . | sha256sum }}
+        checksum/cm: {{ (include (print $.Template.BasePath "/argocd-configs/argocd-cm.yaml") . | fromYaml).data | toYaml | sha256sum }}
```

What about
```suggestion
        checksum/cm: {{ include "argo-cd.config.cm" . | sha256sum }}
```
?

Ref: https://github.com/argoproj/argo-helm/blob/936101865d3090183fc4979546685ece7f78ac23/charts/argo-cd/templates/argocd-configs/argocd-cm.yaml#L15-L17